Dockerfile

```diff
@@ -14,19 +15,17 @@ COPY . /src/panacea-core
 # Set working directory for the 'panacea-core' build
 WORKDIR /src/panacea-core
 
-# Install panacea-core
-RUN make clean && make build
-
 # Get 'libwasmvm.so' from wasmvm
-RUN git clone -b v0.14.0 https://github.com/CosmWasm/wasmvm.git /src/wasmvm
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v0.16.2/libwasmvm_muslc.a /lib/libwasmvm_muslc.a
+RUN sha256sum /lib/libwasmvm_muslc.a | grep 0e62296b9f24cf3a05f8513f99cee536c7087079855ea6ffb4f89b35eccdaa66
+
+RUN make clean && LEDGER_ENABLED=true BUILD_TAGS=muslc make build
 
 # Final image
-FROM debian:buster-slim
```

Comment:
I think it's better to use debian:buster-slim here as before.
Because panacea binary is usally used in Ubuntu.

Reply:
Thanks for your feedback!

I change final image from alpine to ubuntu in Dockerfile.

Thank you :)